integration-tests/environments/react-native-test-app/ios/Podfile.lock

```diff
@@ -1107,7 +1107,7 @@ PODS:
     - React-Core
     - React-jsi
   - ReactTestApp-Resources (1.0.0-dev)
-  - RealmJS (12.7.0-rc.0):
+  - RealmJS (12.7.1):
```

Comment:
I wonder why this was not already on `main` 🤔

Reply:
Probably just didn't get committed 🤷‍♀️